templates/taxi/car_detail.html

```diff
@@ -5,8 +5,8 @@
   <h1>Drivers</h1>
   <hr>
   <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.first_name }} {{ driver.last_name }}</li>
-  {% endfor %}
+    {% for driver in car.drivers.all %}
```

Comment:
The iteration over `car.drivers.all` should use parentheses: `car.drivers.all()`. This is necessary to call the method and retrieve the list of drivers.